test/Microsoft.TestPlatform.AcceptanceTests/ExecutionTests.cs

```diff
@@ -114,6 +114,24 @@ public void TestSessionTimeOutTests(RunnnerInfo runnerInfo)
         }
 
         [CustomDataTestMethod]
+        [NETFullTargetFramework]
```

Comment:
I'd prefer to cover this scenario with unit tests instead of a acceptance test.

Reply:
Sure.. Sent a new PR which is solving some other issue with TestSessionTimeout has a unittest which cover this scenario.